test/Tests.Infrastructure/RavenTestBase.cs

```diff
@@ -62,6 +63,17 @@ protected static void CreateNorthwindDatabase(DocumentStore store)
             store.Maintenance.Send(new CreateSampleDataOperation());
         }
 
+        protected async Task CreateLegacyNorthwindDatabase(DocumentStore store)
+        {
+            using (var stream = Assembly.GetExecutingAssembly().GetManifestResourceStream("Tests.Infrastructure.Data.Northwind.ravendbdump"))
```

Comment:
Maybe rename the file to Northwind.4.2.ravendbdump so we will know from which version the dump is?